### PR TITLE
Add auto-scrolling to Treemap

### PR DESCRIPTION
**Pull request by peterjcoffman**
Hoist P/R Checklist
-------------------

Review and check off the below. Items that do not apply can also be checked off to indicate they
have been considered. If unclear if a step is relevant, please leave unchecked and note in comments.

- [x] Up to date with `develop` branch as of last change.
- [x] Ready for review (or open as a draft PR / add `wip` label).
- [x] Added CHANGELOG entry (or N/A)
- [x] Reviewed for breaking changes (add `breaking-change` label + CHANGELOG if so, or N/A)
- [x] Updated doc comments / prop-types (or N/A)
- [x] Reviewed and tested on Mobile (or N/A)
- [x] Created Toolbox branch / PR (link provided here, or N/A)

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

**Comment by peterjcoffman** [comment]
Relates to this issue:
https://github.com/xh/hoist-react/issues/1510

File: desktop/cmp/treemap/TreeMap.js [hunk 1]
```diff
@@ -302,6 +302,8 @@ class LocalModel {
         this.chart.series[0].data.forEach(node => {
             node.select(toSelect.has(node.id), true);
         });
+
+        gridModel.ensureSelectionVisible();
     }
```
Comment (lbwexler) [inline desktop/cmp/treemap/TreeMap.js]: This should be made null-safe, as it is on 292.  The associated GridModel is optional in TreeMapModel.

**Comment by lbwexler** [comment]
Please see fix to make this null-safe.

Also, `TreeMapModel.propTypes` in file above is bust -- should be `TreeMap.propTypes`.  Can you fix with this change?

Thanks

File: desktop/cmp/treemap/TreeMap.js (hunk 1 above)
Comment (peterjcoffman) [reply desktop/cmp/treemap/TreeMap.js]: Done.

**Pull request merged**